tuf/api/metadata.py

```diff
@@ -336,6 +336,8 @@ def __init__(
         unrecognized_fields: Optional[Mapping[str, Any]] = None,
     ) -> None:
 
+        # "_type" is not actually meant to be "protected" in the OOP sense
+        # but rather is a workaround to not shadow the built-in "type" name.
```

Comment:
I searched for that and one relevant response from Stackoverflow: https://stackoverflow.com/a/10568115/7058471

Reply:
One other reason to use `_type` rather than `type` is that it better matches the spec. 

Historical context: the use of `_type` in the spec was to match the implementation, which did not want to shadow the builtin `type()` method. Shadowing `type()` was more problematic in a procedural implementation, and will likely be less of an issue in the OOP metadata API – but the name in the spec is `_type` until we do something with https://github.com/theupdateframework/specification/issues/149 and a possible 2.0 of the specification.

Reply:
oof I forgot the spec uses _type . Thanks